desktop.css

```diff
@@ -0,0 +1,296 @@
+body {
+  background-image:
+    url("/multimedia/circle-1-header.svg"),
+    url("/multimedia/circle-2-header.svg"),
+    url("/multimedia/circle-4.svg"),
+    url("/multimedia/circle-5.svg"),
+    url("/multimedia/circle-6.svg"),
+    url("/multimedia/circle-7.svg");
+  background-repeat:
+    no-repeat,
+    no-repeat,
+    no-repeat,
+    no-repeat,
+    no-repeat,
+    no-repeat;
+  background-size:
+    300px 300px,
+    325px 325px,
+    330px 1000px,
+    280px 1200px,
+    600px 280px,
+    430px 1000px;
+  background-position:
+    top 198px left 69px,
+    left 1180px top 520px,
+    left -10px top 1150px,
+    right 0 top 1600px,
+    left -100px top 2500px,
+    right -10px top 2450px;
+}
+
+.main-page-header {
+  display: none;
+}
+
+.header-hidden {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #ffff;
+  padding: 0 142px;
+  margin-bottom: 0;
+}
+
+.nav-bar {
+  display: flex;
+  gap: 20px;
+}
+
+.nav-links {
+  display: flex;
+  gap: 20px;
+  font-size: 0.93rem;
+}
+
+.message {
+  display: flex;
+  align-items: center;
+  padding-bottom: 4px;
+  align-content: center;
+}
+
+.work-element-hidden {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #ffff;
+  margin-bottom: 0;
+  padding: 0 40px;
+}
+
+.logo {
+  padding: 10px 0;
+}
+
+.open-menu {
+  padding: 10px 0;
+}
+
+.headline {
+  padding: 130px 142px 114px;
+  gap: 88px;
+}
+
+.name {
+  font-size: 5.62rem;
+  text-align: center;
+  line-height: 105px;
+}
+
+.occupation {
+  font-size: 2rem;
+  margin-bottom: 0;
+  padding: 0;
+  text-align: center;
+  line-height: 47px;
+}
+
+.about-text {
+  margin: 0;
+  padding: 0 10px 45px;
+  text-align: justify;
+  text-align-last: center;
+  line-height: 32px;
+  left: 0.5px;
+  width: 100%;
+}
+
+.about-text-links {
+  padding: 0 354px;
+}
+
+.about-links {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  padding-top: 50px;
+  gap: 23px;
+}
```

Comment:
This part of the headline doesn't quite look like the one from the template design.

Here's your design:

<img width="456" alt="Screen Shot 2022-06-03 at 15 42 10" src="https://user-images.githubusercontent.com/82426682/171949356-65a686cd-ab60-49cb-a926-33898caf32af.png">

Here's the template design:

<img width="460" alt="Screen Shot 2022-06-03 at 15 42 38" src="https://user-images.githubusercontent.com/82426682/171949469-d673aa10-4128-4e60-8951-b0ec6ba97493.png">

Kindly fix this.

Reply:
![image](https://user-images.githubusercontent.com/92395484/171952976-cec6ede4-de06-4bc8-ac58-7dc1865d2182.png) It looks all right to me, I don't understand.

Reply:
It may be a Chrome issue, but this is the way your project is being rendered on my end.

<img width="1440" alt="Screen Shot 2022-06-03 at 16 43 14" src="https://user-images.githubusercontent.com/82426682/171957797-55eb66b9-bd27-4a1d-936e-9dba78e06b17.png">

Reply:
@vechicin What should I do then? What do you recommend?